docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+services:
+
+  polkadot:
+    image: chevdor/polkadot:latest
+    volumes:
+      - "polkadot-data:/data"
+    command: sh -c "polkadot --rpc-external --prometheus-external --log error --rpc-cors=all"
+
+  vedran:
+    command: start --fee=0.1 --auth-secret=test-secret --log-level debug
+    image: nodefactory/vedran:v0.1.1
+    container_name: "vedran"
+
+  vedran-daemon:
+    image: nodefactory/vedran-daemon:v0.1.2
+    depends_on:
+      - vedran
+      - polkadot
+    entrypoint: /bin/sh -c "./vedran-daemon --id test-id --lb http://vedran:4000 --node-rpc http://polkadot:9933 --node-metrics http://polkadot:9615 --payout-address 0xtestaddress --log-level error"
```

Comment:
should be:
command: --id test-id --lb http://vedran:4000 --node-rpc http://polkadot:9933 --node-metrics http://polkadot:9615 --payout-address 0xtestaddress --log-level error